server/src/main/kotlin/suwayomi/tachidesk/manga/controller/DownloadController.kt

```diff
@@ -107,6 +115,27 @@ object DownloadController {
         }
     )
 
+    val queueChapters = handler(
```

Comment:
Missing `pathParam`

Reply:
I don't think it needs one?

Reply:
`/download/batch/` does not have any `pathParam`.

Reply:
oh, my bad